dbms/src/Flash/Executor/toRU.cpp

```diff
@@ -24,9 +24,7 @@ UInt64 toCPUTimeMillisecond(UInt64 cpu_time_ns)
     if (unlikely(cpu_time_ns == 0))
         return 0;
 
-    double cpu_time_millisecond = static_cast<double>(cpu_time_ns) / 1'000'000L;
-    auto ceil_cpu_time_millisecond = ceil(cpu_time_millisecond);
-    return ceil_cpu_time_millisecond;
+    return static_cast<double>(cpu_time_ns) / 1'000'000L;
```

Comment:
Why modify here, because you want to round down?

Reply:
https://en.cppreference.com/w/c/language/conversion#Usual_arithmetic_conversions
1'000'000L will convert to double implicitly.

Reply:
But the return type is UInt64 of `toCPUTimeMillisecond`, and then double value will be rounded down